examples/react-urql-retry/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "urlq-retry-query",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/react": "^11.2.6",
+    "@testing-library/user-event": "^12.8.3",
```

Comment:
nit: Extraneous dependencies; also as the other one, `web-vitals` is also extraneous and `react-scripts` to be a minor range and to go to `devDependencies`